charts/istioconfig/templates/service-authorization.yaml

```diff
@@ -0,0 +1,18 @@
+# Sample AuthZ Policy for future istio.
```

Comment:
What do you mean by future Istio here? 1.4 that we just upgraded to or something later?

Reply:
ahh, sorry, forgot to change that